en/appendices/4-2-migration-guide.rst

```diff
@@ -44,6 +44,10 @@ Database
 ORM
 ---
 
+- ``TableLocator::allowFallbackClass()`` was added. This method lets you
+  disable automatically generated fallback table classes. This feature is
+  currently an opt-in feature but will become a enabled by default in the
```

Comment:
I think it is the opposite: It is currently opt-out by default (enabled always by default), and will require opt-in in the future.

Reply:
I meant to say that disabling auto-tables is opt-in though.